src/core/hle/service/fs/fs_user.cpp

```diff
@@ -816,7 +837,79 @@ void FS_USER::GetSaveDataSecureValue(Kernel::HLERequestContext& ctx) {
     rb.Push<u64>(0);      // the secure value
 }
 
-void FS_USER::Register(u32 process_id, u64 program_id, const std::string& filepath) {
+void FS_USER::SetThisSaveDataSecureValue(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp(ctx);
+    u32 secure_value_slot = rp.Pop<u32>();
```

Comment:
```suggestion
    const u32 secure_value_slot = rp.Pop<u32>();
```
ditto for other rp.Pop calls